doccano_client/beta/controllers/example.py

```diff
@@ -126,8 +138,7 @@ def create(self, example: Example) -> ExampleController:
         """Upload new example for Doccano project, return the generated controller
 
         Args:
-            example: Example. The only fields that will be uploaded are text, annnotations,
-                and meta.
+            example: Example. See mock_api_responses for the desired format.
```

Comment:
Same as previously: If it's not too much to ask might good to add the format here quickly, saving future contributors/readers of code from having to look at another file.